pipeline/utils/task_utils.py

```diff
@@ -36,6 +36,13 @@ def gradle_task(toolkit_path, task_name, task_args):
             task_name, '-Pclargs=' + ','.join(task_args)]
 
 
+def api_name(short_name, version, is_cloud):
```

Comment:
Using a boolean for "is_cloud" is not scalable - it is essentially guaranteed to change as we handle more organizations. I would much prefer an organization identifier, which for this case would be "google_cloud".